deps/aa_tree.c

```diff
@@ -1,21 +1,23 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. 
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  *
  *    Copyright 2020 (c) Fraunhofer IOSB (Author: Julius Pfrommer)
  */
 
 #include "aa_tree.h"
 #include <stddef.h>
 
-#if !defined(_MSC_VER) || _MSC_VER >= 1800
-# include <inttypes.h>
-#elif !defined(uintptr_t)
-  /* Workaround missing standard includes in older Visual Studio */
-# if !defined(__midl) && (defined(_X86_) || defined(_M_IX86)) && _MSC_VER >= 1300
+#ifndef UNDER_CE
```

Comment:
That's a deep nesting of #ifdefs and a bit confusing.
I think it could be a flat list. As in
``` 
#ifdef UNDER_CE

#elif !defined(_MSC_VER) ....

#else

#endif
```

Reply:
Changed it as you suggested and added a comment for `UNDER_CE` since there's nothing to define/include in that case.